ouroboros-consensus-protocol/ouroboros-consensus-protocol.cabal

```diff
@@ -76,7 +76,7 @@ library
     containers,
     mtl,
     nothunks,
-    ouroboros-consensus >=0.23 && <0.25,
```

Comment:
Maybe we will need another patch release for this due to changes in testing libraries?